website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -493,3 +493,4 @@ Also see the [advanced topics][advanced_topics] doc for more use cases and under
 [multigroup-kubebuilder-doc]: https://book.kubebuilder.io/migration/multi-group.html
 [quickstart-bundle]:/docs/olm-integration/quickstart-bundle
 [doc-olm]:/docs/olm-integration/quickstart-bundle/#enabling-olm
+[kube-builder]: https://book.kubebuilder.io/quick-start.html#installation
```

Comment:
See the comment https://github.com/operator-framework/operator-sdk/pull/4395/files#r560166486. IMO it should not be added at all.

Reply:
Removed as per  #4395 (comment).